README.md

```diff
@@ -1,54 +1,281 @@
+New users on Windows should click the "prereqs" button too, see *Installing* for info.
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+**Never mix different versions** of BizHawk — Keep each version in its own folder.
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+Jump to:
+* Installing
+	* [Windows 7/8.1/10](https://github.com/TASVideos/BizHawk/blob/master/README.md#windows-78110)
+	* [GNU+Linux and macOS](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos)
+* Building
+	* [Windows 7/8.1/10](https://github.com/TASVideos/BizHawk/blob/master/README.md#windows-78110-1)
+	* [GNU+Linux and macOS](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos-1)
+* [Usage](https://github.com/TASVideos/BizHawk/blob/master/README.md#usage)
+	* [TASing](https://github.com/TASVideos/BizHawk/blob/master/README.md#tasing)
+	* [Cores](https://github.com/TASVideos/BizHawk/blob/master/README.md#cores)
+* [Support and troubleshooting](https://github.com/TASVideos/BizHawk/blob/master/README.md#support-and-troubleshooting)
+* [Contributing](https://github.com/TASVideos/BizHawk/blob/master/README.md#contributing)
+	* [Testing](https://github.com/TASVideos/BizHawk/blob/master/README.md#testing)
+* [Related projects](https://github.com/TASVideos/BizHawk/blob/master/README.md#related-projects)
+* [License](https://github.com/TASVideos/BizHawk/blob/master/README.md#license)
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+## Features and systems
 
-## Supported Systems
+The BizHawk common features (across all cores) are:
+* format, region, and integrity detection for game images
+* 10 save slots with hotkeys and ∞ named savestates
+* speed control, including frame stepping and rewinding
+* memory view/search/edit in all emulated hardware components
+* input recording (making TAS movies)
+* screenshotting and recording audio + video to file
+* firmware management
+* input, framerate, and more in a HUD over the game
+* emulated controllers via a comprehensive input mapper
+* Lua control over core and frontend (Windows only)
+* hotkey bindings to control the UI
 
- * Nintendo Entertainment System / Famicom / Famicom Disk System (NES/FDS)
- * Super Nintendo (SNES)
- * Nintendo 64
- * Game Boy, Game Boy Color, and Super Game Boy
- * Game Boy Advance
- * Sony PlayStation
- * Sega Master System, Game Gear, and SG-1000
- * Sega Genesis / Sega-CD
- * Sega Saturn
- * PC-Engine (TurboGrafx-16) / CD-ROM & SuperGrafx 
- * Atari 2600
- * Atari 7800
- * Atari Lynx
- * ColecoVision
- * TI-83 Calculator
- * Wonderswan and Wonderswan Color 
- * Apple II
- * Commodore 64
- * Sinclair ZX Spectrum
+Supported consoles and computers:
 
-## Resources
+* Apple II
+* Atari 2600
+* Atari 7800
+* Atari Lynx
+* Commodore 64
+* ColecoVision
+* Game Boy / Color
+* Game Boy Advance
+* IntelliVision
+* N64
```

Comment:
Let's use full names here. Nintendo 64, (Super) Nintendo Entertainment System, also we emulate Famicom Disk System.

Reply:
done

Reply:
You added manufacturer and sublists, nice.